ansible/provision-private-proxied-cluster-airgap-deb.yml

```diff
@@ -0,0 +1,65 @@
+### This is not a truly airgapped cluster!!! The intent here is specifically to test the install flow for an airgapped install
+
+---
+- name: Provision proxy on client
+  hosts: client
+  vars:
+    squid_acl_localnet:
+      - "10.0.1.0/24"
+  tasks:
+    - name: Install and configure proxy on client
+      ansible.builtin.include_role:
+        name: mrlesmithjr.squid
+
+- name: Bundle DEBs
+  hosts: client
+  vars:
+    basearch: "amd64" # must match the output of `uname -m`. however on ubuntu you need to use the debian version of the output. So for example x86_64 is amd64
+    os_distribution_major_version: "focal" # must match the output of `. /etc/os-release && echo ${VERSION_ID%%.*}`
+    os_distribution: "ubuntu" # must match the output of `. /etc/os-release && echo $ID`
+    redpanda_version: "23.1.13-1" # note that the -1 is the build number for this version. it is typically -1 but if the version is incremented it will be higher
+    rpm_or_deb: "deb"
+    ansible_ssh_common_args: "-o ProxyCommand='ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i /app/artifacts/testkey -W %h:%p -q {{ hostvars[groups['client'][0]].ansible_user }}@{{ hostvars[groups['client'][0]].inventory_hostname }}'"
```

Comment:
is this only needed for testing? if so, can we figure out how to handle the StrictHostKeyChecking and UserKnownHostsFile just for that? This is going to raise some eyebrows in some secure environments.

Reply:
I would recommend against using it for a secure environment yes, but for ansible it does need to be set -- especially for CI. I really don't think users in secure environments should consume these playbooks without heavy customization. They're intended to demonstrate features and provide a testing bed for CI rather than providing a 1:1 for customers.